### ISSUE TITLE
Exploratory Exercise 02 - Compare source of radas

### ISSUE THREAD

**Issue by andkov** [kse-ua/ua-de-center#7]
# Problem

Please see "./analysis/task/02-compare-rada-sources" for detailed description of the problem and fuller data context.

```r
# We have two data files mapping radas to hromadas
# The first dataset we called  `rada_local` comes from the file
# "Центр суспільних даних. Місцеві ради 2014"  https://docs.google.com/spreadsheets/d/1iEbUsZSDGbJUzl_6wC3vgoVJ7GzOlc9f/edit?usp=sharing&ouid=106674411047619625756&rtpof=true&sd=trueентр суспільних даних. Місцеві ради 2014
rada_local
#  this data set stores information on N = _______ radas
rada_local %>% pull(rada_code) %>% unique() %>% length() %>% scales::comma()

#  the SECOND dataset we called `rada_united` comes from the file
# Центр суспільних даних. Обєдання громад - https://docs.google.com/spreadsheets/d/1xAFUDx8nf2oaIezWSBLaqitdxwEiQaOw/edit?usp=sharing&ouid=106674411047619625756&rtpof=true&sd=true
rada_united
# the list of radas in this file counts N = ________ radas
rada_united %>% pull(rada_code) %>% unique() %>% length() %>% scales::comma()
# this file records what radas makes up hromadas at the end of the amalgamation (2021)
# TODO:
# Explore the discrepancy between these two files
# Using the labels in the dataset `ds_admin`,  describe what radas/hromadas are
# missing from each file and speculate/expolain why. 

# Answer the following question:
# If we disregard the "Local" source and use only rada_united, will we miss anything relevant to our project? 
# In other words, if we need to rely on the mapping between radas and hromadas,
# are we safe to use the mapping derived from the "United" source? ( I think yes,
# but we need the proof)

# Notes:
# 1. Occupied territories is the most likely culprit, but there might be something else 
# 2. The report should compile into an html document
# 3. Please use the "main" branch, but create a separate script with your solution
# and call it "./analysis/tasks/02-compare-rada-sources-yourname.R"
```
